modules/nf-core/antismash/antismashdownloaddatabases/tests/nextflow.config

```diff
@@ -0,0 +1,5 @@
+process {
+    withName: ANTISMASH_ANTISMASHDOWNLOADDATABASES {
+        memory = 7.GB
```

Comment:
Are the 7GB required just to download the db?

Reply:
IIRC Yes as it does some extra post-processing things...